src/async_fuse/fuse/file_system.rs

```diff
@@ -283,4 +284,60 @@ pub trait FileSystem {
 
     /// Set fuse fd into `FileSystem`
     async fn set_fuse_fd(&self, fuse_fd: RawFd);
+
+    /// Stop all async tasks
+    fn stop_all_async_tasks(&self);
```

Comment:
Shall we move it into another trait? I don't think all the FileSystems have async tasks.